auth-api/migrations/versions/48bd1f6e6c99_.py

```diff
@@ -0,0 +1,362 @@
+"""empty message
+
+Revision ID: 48bd1f6e6c99
+Revises: 8d3f85a7dfb3
+Create Date: 2019-07-18 14:48:26.535116
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "48bd1f6e6c99"
+down_revision = "8d3f85a7dfb3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "entity",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("coop_entity_id", sa.Integer(), nullable=False),
```

Comment:
Agreed - I'll switch to 'business_identifier'.  In the legal-api, the primary key for 'business' is an Integer.  Should I still switch it to String in case this changes?

Also, I will extract an address model out as you and @pwei1018 suggest.

Thanks for the feedback!